geopy/geocoders/googleplaces.py

```diff
@@ -0,0 +1,217 @@
+"""
+:class:`.GooglePlaces` is the Google Places API.
+"""
+from geopy.compat import urlencode
+from geopy.geocoders.base import Geocoder, DEFAULT_TIMEOUT, DEFAULT_SCHEME
+from geopy.exc import (
+    GeocoderQuotaExceeded,
+    GeocoderQueryError,
+)
+from geopy.location import Location
+
+
+__all__ = ("GooglePlaces", )
+
+
+class GooglePlaces(Geocoder):  # pylint: disable=R0902
+    """
+    Geocoder using the Google Places API. Documentation at:
+        https://developers.google.com/places/documentation/
+    """
+
+    def __init__(
+            self,
+            api_key=None,
+            domain='maps.googleapis.com',
+            scheme=DEFAULT_SCHEME,
+            timeout=DEFAULT_TIMEOUT,
+            proxies=None,
+            user_agent=None
+    ):  # pylint: disable=R0913
+
+        """
+        Initialize a customized Google geocoder.
+
+        API authentication is only required for Google Maps Premier customers.
+
+        :param string api_key: The API key required by Google to perform
+            geocoding requests. API keys are managed through the Google APIs
+            console (https://code.google.com/apis/console).
+
+        :param string domain: Should be the localized Google Maps domain to
+            connect to. The default is 'maps.googleapis.com', but if you're
+            geocoding address in the UK (for example), you may want to set it
+            to 'maps.google.co.uk' to properly bias results.
```

Comment:
have you tested this behaviour? (i know it's true for the geocoder, i'm not sure it's true for autocomplete as well)